### ISSUE TITLE
Feature Request: Bulk add learners to TTT event

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#1566]
Currently we have the ability to add learners to any event via the "Bulk add persons" feature.  However for Instructor Training (TTT) events we also need to add in the member site the learner is associated with or to mark them as an open applicant.  It would be useful to have a bulk add feature that adds in a column for membership (or open) to make it easier to add people in.

**Comment by pbanaszkiewicz** [comment]
This sounds like an extension to bulk-add persons interface, unless you want an entirely separate interface for that. Please let me know.

**Comment by maneesha** [comment]
Yes, I think this would extend the existing feature, so we could include data associated with learners at instructor training events.

**Comment by maneesha** [comment]
This was part of a pre-pandemic project to restructure and streamline the instructor training checkout process. We are tabling this until we discuss more internally.
